File: Practice problem/Harshad number/Harshad number.py
# 연습문제 하샤드 수 Level 1

def solution(x):
    z = x
    y = 0
    while z > 0:
        y += z%10
        z -= z%10
        z /= 10
    if ((x % y) == 0 ):
        return True
    else:
        return False

File: Practice problem/Harshad number/Harshad number.java
// 연습문제 하샤드 수 Level 1

class Solution {
    public boolean solution(int x) {
        int z = x;
        int y = 0;
        while(x > 0) {
            y += x%10;
            x /= 10;
        }
        if(z % y == 0){ return true; }
        return false;
    }
}
